.github/workflows/ci.yml

```diff
@@ -159,14 +167,15 @@ jobs:
           # Check with all features.
           # (But watch out for backend-selection features in wgpu-core; some of
           # those only build on the right platforms.)
+          cargo clippy --target ${{ matrix.target }} -p wgpu-hal --tests --all-features
+          cargo clippy --target ${{ matrix.target }} -p wgpu-core --tests --all-features
           cargo clippy --target ${{ matrix.target }} -p wgpu -p wgpu-info -p player --tests --all-features
```

Comment:
I think this can now be one call to clippy with all the crates, as --all-features works - that's the only reason we didn't do it all in one command before

Reply:
Done.